app-client/src/main/resources/application-docker.properties

```diff
@@ -0,0 +1,12 @@
+#SERVER CONFIGURATIONS
+server.port= 8081
+server.servlet.context-path= /cdc
+
+#SPRING SSE TIMEOUT
+spring.mvc.async.request-timeout= -1
+
+#USERS URL
+users.url= http://cdc-service:8082/api/users
+
+#URL TO STREAM CHANGES FROM DEBEZIUM
+events.streaming.url= http://cdc-service:8082/api/users/stream
```

Comment:
Which service is `cdc-service`? Service name in `docker-compose.yaml` is `app`

Reply:
Resolved.
That is rather the name of the container.
My bad!